compiler/AST/astutil.cpp

```diff
@@ -252,16 +251,23 @@ void compute_call_sites() {
     else
       fn->calledBy = new Vec<CallExpr*>();
   }
+
   forv_Vec(CallExpr, call, gCallExprs) {
-    if (FnSymbol* fn = call->isResolved()) {
+    if (call->isEmpty() == true) {
+
+    } else if (FnSymbol* fn = call->isResolved()) {
```

Comment:
That's a good question; I'm not certain how that's going to turn out.

Right now compute_call_sites() is used during resolution and optimization.  I had to
add the isEmpty() to protect isResolved(), AKA resolvedFunction(), from empty call exprs.
I think for now I am "ok" with this being resolvedFunction() but that might change as I
understand things better.